docs/source/conf.py

```diff
@@ -117,6 +117,9 @@
     (r"py:.*", r".*np.ndarray.*"),
     (r"py:.*", r".*numpy._typing._array_like._ScalarType_co.*"),
     (r"py:.*", r".*idex.l1a.TRIGGER_DESCRIPTION.*"),
+    (r"py:.*", r".*.spice.geometry.SpiceBody.*"),
```

Comment:
More nitpick ignores for IntEnums and some `numpy_typing`. What confuses me is that theses sphinx warnings would go away if I built the documentation a second time without cleaning. I simply couldn't figure out how to fix these issues without just adding the ignore regex here.